packages/studio-ui/.size-limit.cjs

```diff
@@ -1,7 +1,7 @@
 module.exports = [
   {
     path: "lib/**/*.js",
-    limit: "850 kB",
+    limit: "870 kB",
```

Comment:
is this bump because of the new `platform` dep?

Reply:
Yep!

Reply:
that's right! Before the size was around 846 kB, but with the new dep it's around 868 kB.

Reply:
Got it! It's not a big bump, but out of curiosity, is there an import we can use to narrow down what part of `platform` we import? (e.g. like `lodash/isEqual` instead of `lodash`)

Reply:
not that I could find, unfortunately